common/static/js/src/string_utils.js

```diff
@@ -45,4 +47,277 @@
         return _.template(text, {interpolate: /\{(.+?)\}/g})(values);
     };
     this.interpolate_text = interpolate_text;
+
+    edx.StringUtils = (function() {
```

Comment:
Reviewers: I put this into string_utils as it already exists and is available in both Studio and LMS. Ultimately we need to find a way to load the UI Toolkit classes asynchronously and then have them made available in the edx namespace in the same way, but that seems difficult because LMS and Studio behave differently, and because we need to ensure that the client code is never invoked before the files have completed loading.

@tobz may have a good insight as to how to do this.